packages/fxa-shared/metrics/amplitude-event.1.schema.json

```diff
@@ -257,9 +257,11 @@
         "required": ["language"],
         "properties": {
           "user_properties": {
+            "type": "object",
```

Comment:
We were getting some warnings from AJV during validation without specifying this in several places.